components/auth/magic-word-form.tsx

```
@@ -0,0 +1,213 @@
+            </div>
+            <p className="text-xs text-gray-500 mt-1">
+              Share this word with friends to join the same room
+            </p>
+          </div>
+
+          {/* Name Input */}
+          <div>
+            <label className="block text-sm font-medium text-cyan-300 mb-2">
+              Your Name
+            </label>
+            <Input
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)